src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.VisualStudioTeamServices.cs

```diff
@@ -98,7 +98,10 @@ private static GitPullRequest CreatePullRequest(string sourceBranch, string targ
             foreach (var build in builds)
             {
                 var artifacts = await buildClient.GetArtifactsAsync(build.Project.Id, build.Id, cancellationToken);
-                if (artifacts.Any(a => a.Resource != null && a.Resource.Data != null && a.Resource.Data.Contains(Options.BuildDropPath)))
+                // A valid artifact would:
+                // 1. Have a resource
+                // 2. The resource would either point to a cpvsbuild path (or) be uploaded to the artifacts service with a build number label.
+                if (artifacts.Any(a => a.Resource != null && a.Resource.Data != null && (a.Resource.Data.Contains(Options.BuildDropPath) || a.Resource.Data.Contains(build.BuildNumber))))
```

Comment:
Does this imply our artifact name has to be the build number? We actually *shouldn't* be doing that IIRC. I know we have repos configured that way, but it's actually broken. :smile:

Reply:
oh, derived that from other places in code. I'm open to change it though. One option is to tie this with `droppath` and pass in a `server` in there. Thoughts?